mod_deploy/controllers.py

```diff
@@ -135,6 +136,11 @@ def deploy():
         with open('build_commit.py', 'w') as f:
             f.write(build_commit)
 
+        # Update runCI
+        runCIrepo = os.path.join(repo,'install/ci-vm/ci-linux/ci/runCI')
```

Comment:
No, `repo` is not a path string.

Reply:
Also, please split up the other string. We're using path.join, so use it for all parts.